README.md

```diff
@@ -1 +1,39 @@
-##Network Service
+# Network Service 
+
+![macOS](https://img.shields.io/badge/macOS-13.3+-blue) ![Xcode](https://img.shields.io/badge/Xcode-14.3+-red)  ![Xcode](https://img.shields.io/badge/Xcode-14.3+-red)  ![Swift](https://img.shields.io/badge/Swift-5.8-orange) ![iOS](https://img.shields.io/badge/iOS-15%2B-lightgrey)
+
+The repo contains a Swift Package with the implementation of the HTTP network service for the iOS platform.
+
+⚠️ The Network service is under active development.
+
+## Table of contents
+
+* [Installation](#installation)
+* [Authors](#authors)
+
+## Installation
+
+**Pre-requisites**: _installed Xcode with the version specified under the [current dev environment](#current-dev-environment)._
+
+To install Network Service package via Xcode
+
+1. Go to `File` -> `Add Packages`
```

Comment:
nit:

Instead of the first items 1 and 3, you can share the link to the official documentation.

Reply:
added link to official documentation